fdm-app/app/routes/welcome.tsx

```
@@ -0,0 +1,308 @@
+                            <Button variant={"outline"}>
+                                {`Lees meer over ${clientConfig.name}`}{" "}
+                                <MoveDown />
+                            </Button>
+                        </div> */}
+                    </div>
+                </div>
+                <div className="hidden bg-muted lg:block">
+                    <img
+                        src="
+https://images.unsplash.com/photo-1625565570971-e6b404974366?q=80&w=1930&auto=format&fit=crop&ixlib=rb-4.1.0&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D"
+                        alt='Photo by <a href="https://unsplash.com/@sonysnaper?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Rickie-Tom Schünemann</a> on <a href="https://unsplash.com/photos/herd-of-cow-on-green-grass-field-during-daytime-MX_wmmlo2Fs?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>'
```

Biome (1.9.4)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)